opm/autodiff/FlowMainEbos.hpp

```diff
@@ -410,17 +409,13 @@ namespace Opm
             outputDoublePrecisionParam += restart_double_si ? "true" : "false";
             argv.push_back(outputDoublePrecisionParam.c_str());
 
-#if defined(_OPENMP)
             std::string numThreadsParam("--threads-per-process=");
-            int numThreads = omp_get_max_threads();
-
+            int numThreads = 1; // TODO
             numThreadsParam += std::to_string(numThreads);
```

Comment:
Does this mean that thread parallelization will not be available for the assembly in flow if this gets merged as is.  
Why not keep numThreads = omp_get_max_threads(), or just add a parameter for this flow. 
To test the new thread parallelization I need to be able to run Flow with it.

Reply:
> Does this mean that thread parallelization will not be available for the assembly in flow if this gets merged as is.

yes. the plan is that the user needs to specify the number of threads explicitly using a command line parameter. If wanted, I'll add this..

> Why not keep numThreads = omp_get_max_threads(), or just add a parameter for this flow.
To test the new thread parallelization I need to be able to run Flow with it.

the problem is that this kills the MPI parallelization because if always spawning the maximum number of worker threads, running an OpenMP enabled `flow` with MPI on a machine with N cores will result in N processes that each exhibit N threads. I severely doubt that this is peformant for N != 1.